modules/book.js

```diff
@@ -0,0 +1,4 @@
+export default function Book(title, author) {
+  this.title = title;
+  this.author = author;
+}
```

Comment:
- [ ] Kindly convert this to ES6 function (arrow function) as this is a practice to use ES6 syntax 😇